e2e/cypress/integration/team_settings/invite_members_spec.js

```diff
@@ -11,43 +11,65 @@
 // Group: @team_settings
 
 function openClickInviteMenuItem() {
-    // * validating the side bar is visible
+    // * Verify the side bar is visible
     cy.get('#sidebarHeaderDropdownButton').should('be.visible');
 
-    // # clicking on the side bar
+    // # Click on the side bar
     cy.get('#sidebarHeaderDropdownButton').click();
 
-    // * Team settings button must be visible
+    // * Verify team settings button is visible
```

Comment:
Should this comment be `Verify Invite People button is visible` instead of 'team settings'?